spec/hashTable.js

```diff
@@ -0,0 +1,171 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable'
+
+chai.use( chaiChange )
+
+describe( 'HashTable', () => {
+
+	it( 'exists', () => {
+		expect( HashTable ).to.be.a( 'function' )
+	})
+
+	let myHashTable, testArray
+
+	beforeEach( () => {
+		myHashTable = new HashTable()
+	})
+
+	context( '#put( key, value )', () => {
+
+		it( 'should add the key value pair to the Hash Table', () => {
+			expect( () => myHashTable.put( "name", "Steve" ) ).to.alter( () => myHashTable.size(), { by: 1 } )
+			expect( myHashTable.contains( "name" ) ).to.be.true
+			expect( myHashTable.get( "name" ) ).to.eql( "Steve" )
+		})
+
+		it( 'should throw an error if not given two parameters', () => {
+			expect( () => myHashTable.put( "name" ) ).to.throw( Error, "Missing necessary parameter(s)" )
+		})
+
+		it( 'should throw an error if the first argument is not a string', () => {
+			expect( () => myHashTable.put( 9, "train" ) ).to.throw( Error, "Key must be a string" )
+		})
+
+	})
+
+	context( '#get( key )', () => {
+
+		it( 'should return the value associated with the key', () => {
+			myHashTable.put( "firstName", "Steve" )
+			myHashTable.put( "lastName", "Weber" )
+			expect( myHashTable.get( "lastName" ) ).to.eql( "Weber" )
+			expect( myHashTable.get( "firstName" ) ).to.eql( "Steve" )
+		})
+
+		it( 'should return null if key does not exist', ()  => {
```

Comment:
if you are using the word "if" in an `it` description you are likely missing a `context`